frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -478,6 +485,14 @@ public int getNumberOfGpu() {
         return getIntProperty(TS_NUMBER_OF_GPU, 0);
     }
 
+    public boolean getModelControlMode() {
+        String mode = getProperty(MODEL_CONTROL_MODE, "none");
+        if (modelMode == null) {
```

Comment:
To avoid getting the property getProperty should be inside the condition.

Reply:
done